src/ripple/app/tests/MultiSign.test.cpp

```diff
@@ -394,6 +412,209 @@ class MultiSign_test : public beast::unit_test::suite
         BEAST_EXPECT(env.seq(alice) == aliceSeq + 1);
     }
 
+    void test_regularSignersUsingSubmitMulti()
+    {
+        using namespace jtx;
+        Env env(*this, features(featureMultiSign));
+        Account const alice {"alice", KeyType::secp256k1};
+        Account const becky {"becky", KeyType::ed25519};
+        Account const cheri {"cheri", KeyType::secp256k1};
+        env.fund(XRP(1000), alice, becky, cheri);
+        env.close();
+
+        // Attach signers to alice.
+        env(signers(alice, 2, {{becky, 1}, {cheri, 1}}), sig (alice));
+
+        // Give everyone regular keys.
+        Account const beck {"beck", KeyType::secp256k1};
+        env(regkey (becky, beck));
+        Account const cher {"cher", KeyType::ed25519};
+        env(regkey (cheri, cher));
+        env.close();
+
+        // Disable cheri's master key to mix things up.
+        env(fset (cheri, asfDisableMaster), sig(cheri));
+        env.close();
+
+        auto const baseFee = env.current()->fees().base;
+        std::uint32_t aliceSeq;
+
+        // these represent oft-repeated setup for input json below
+        auto setup_tx = [&]() -> Json::Value {
```

Comment:
It looks like the `setup_tx` lambda is effectively a `Json::Value` used as initialiser.
If so, consider making it a  `Json::Value` rather than a lambda.
I like the idea to use an immediately-invoked lambda to directly initialise it.
That way, it could be made const  `Json::Value` (though const is not needed here):

```C++
Json::Value const setup_tx = [] {
     Json::Value jv;
     jv[jss::tx_json][jss::Account] = Account("alice").human();
     ...
     return jv;
}();
```
Account alice is expanded to avoid need to capture.
Also removed parentheses and trailing return fwiw.